ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -88,17 +88,15 @@ public NetworkClient(Selector selector, NetworkConfig networkConfig, NetworkMetr
    * @param pollTimeoutMs the poll timeout.
    * @return a list of {@link ResponseInfo} representing the responses received for any requests that were sent out
    * so far.
-   * @throws IOException if the {@link Selector} associated with this NetworkClient throws
```

Comment:
I'm not sure this should be removed. I remember a discussion about how `IOException` from a `Selector` is fatal and cannot be recovered from - this is not a per operation error (which is what we are trying to isolate?)